blackduck/HubRestApi.py

```diff
@@ -929,6 +935,30 @@ def delete_project_version_by_name(self, project_name, version_name, save_scans=
                 logging.debug("Did not find version with name {} in project {}".format(version_name, project_name))
         else:
             logging.debug("Did not find project with name {}".format(project_name))
+    
+    def delete_project_by_name(self, project_name):
```

Comment:
would it make sense to include a flag for saving scans? with the default being not to save them? e.g. save_scans=False